README-yearnBorg.md

```diff
@@ -104,12 +114,12 @@ New Modules grant `ychad.eth` privileges to bypass Guards restrictions, therefor
 In exceptional circumstances, `ychad.eth` can propose the removal of the Guard via [Co-approval Workflows](#co-approval-workflows).
 Upon DAO co-approval and execution, `ychad.eth` will no longer face any restriction on administrative operations.
 
-**⚠️ Warning**: Disabling a Module revokes `ychad.eth`'s priviledges. In particular, disabling `SudoImplant` will permanently eliminate `ychad.eth`'s ability to add new Modules or remove Guards.
+**⚠️ Warning**: Disabling a Module revokes `ychad.eth`'s privileges. In particular, disabling `SudoImplant` will permanently eliminate `ychad.eth`'s ability to add new Modules or remove Guards. [discuss]
```

Comment:
We could eliminate SudoImplant's ability to disable modules, preventing potential user errors, while `ychad.eth` retains the ability to do so by removing Guard

Reply:
Decided to go for (1), review in process